panel/layout.py

```diff
@@ -129,19 +129,25 @@ def _get_model(self, doc, root=None, parent=None, comm=None):
     def __getitem__(self, index):
         return self.objects[index]
 
+    def __len__(self):
```

Comment:
Shouldn't `pane = Pane(pane, _temporary=True)` in _get_objects above be `pane = create(pane, _temporary=True)`?

Reply:
Actually no, this was deliberate.

Reply:
Nevermind, it wasn't.